bip-XXXX.mediawiki

```diff
@@ -0,0 +1,118 @@
+<pre>
+  BIP: XXXX
+  Layer: Applications
+  Title: DNS Payment Instructions
+  Author: Matt Corallo <bipxxx@bluematt.me>
+          Bastien Teinturier <bastien@acinq.fr>
+  Comments-Summary: No comments yet.
+  Status: Draft
+  Type: Standards Track
+  Created: 2024-02-10
```

Comment:
The preamble is missing mandatory headers. Please add `Comments-URI` and `License`. If this BIP has meanwhile been posted to the mailing list, it would also be great if you could add a link to that and the delving discussion per the `Post-History` header.